src/dos/dos_keyboard_layout.cpp

```diff
@@ -1322,9 +1323,59 @@ void DOS_KeyboardLayout_ShutDown(Section* /*sec*/) {
 	delete test;	
 }
 
+std::map<std::string, int> country_code_map {
```

Comment:
Do we intend to make changes to this map during runtime?
(if not, let's make it read-only).

Reply:
Done.